### PR TITLE
CI: Implement CI publishing pipeline

### PR DESCRIPTION
**Pull request by mmmoore1313**

### CODE REVIEW

File: .github/workflows/bumpVdev.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: "Bump Version"
+
+on:
+  push:
+    branches:
+      - "dev"
```
Comment (jtorreggiani) [inline .github/workflows/bumpVdev.yml:6]: I think in most of these fields the double quotes are redundant.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "spec-ods",
-  "version": "0.1.0",
+  "version": "0.1.2",
```
Comment (jtorreggiani) [inline package.json:3]: Where is `0.1.1`?

**Review by jtorreggiani — CHANGES_REQUESTED** [review]
Could we add a useful title and description so if we need to go back to review this PR in the future, in the case we want to set something up similar, we can find the PR by the title.

Something like "Implement CI publishing pipeline" or similar.

File: package.json (hunk 2 above)
Comment (mmmoore1313) [reply package.json]: `0.1.1` may have been lost in the testing; however, we can reset the version when we choose for future updates. When using `patch`, `minor`, or `major` in the commits, the version is auto updated on push/PR

**Comment by mmmoore1313** [comment]
- Title changed to: Implement CI publishing pipeline
- Quotations removed from branch names in the version bump files

**Review by jtorreggiani — APPROVED** [review]
LGTM ✅

**Pull request merged**